test/login.ts

```diff
@@ -25,8 +26,8 @@ describe('hello query tests', function () {
     }
   `
 
-  it('Should return a user and a token as response', async function () {
-    const createdUser = {
+  it('Should return a user and a  7 day token as response', async function () {
```

Comment:
```suggestion
  it('Should return a user and a long expiration token as response', async function () {
```

Os 7 dias são um dado que não precisa ser fixo. Imporatante é sabermos que existe um tempo de expiração curto e um longo, mas os valores em si não são tão importantes a ponto de ficarem citados em código ou enunciado de testes, certo? Se vc precisar trocar de 7 dias para 14, por exemplo, teria que editar esse enunciado pra ficar coerente.